### PR TITLE
Added information about the four sub-components of Security component

### PR DESCRIPTION
**Pull request by javiereguiluz**
| Q | A |
| --- | --- |
| Doc fix? | no |
| New docs? | yes |
| Applies to | 2.6+ |
| Fixed tickets | #4151 |

### CODE REVIEW

File: components/security/introduction.rst [hunk 1]
```diff
@@ -21,6 +21,17 @@ You can install the component in 2 different ways:
...
 
+Starting from Symfony 2.4, the Security component is divided into four smaller
+sub-components which can be used separately:
+
+* ``symfony/security-acl``, it provides a fine grained permissions mechanism
+  based on Access Control Lists.
+* ``symfony/security-core``, it provides all the common security features, from
+  authentication to authorization and from encoding passwords to loading users.
+* ``symfony/security-csrf``, it provides protection against `CSRF attacks`_.
+* ``symfony/security-http"``, it integrates the core sub-component with the
```
Comment (WouterJ) [inline components/security/introduction.rst]: I would like to reorder this to `core`, `http`, `csrf`, `acl`

Comment (WouterJ) [inline components/security/introduction.rst]: and what about using a definition list?

Comment (javiereguiluz) [inline components/security/introduction.rst]: Much better! Thanks for the suggestion.

File: components/security/introduction.rst [hunk 2]
```diff
@@ -21,6 +21,20 @@ You can install the component in 2 different ways:
...
 
+Starting from Symfony 2.4, the Security component is divided into four smaller
+sub-components which can be used separately:
+
+``symfony/security-core``
+    It provides all the common security features, from authentication to
+    authorization and from encoding passwords to loading users.
+``symfony/security-http``
+  It integrates the core sub-component with the HTTP protocol to handle HTTP
+  requests and responses.
```
Comment (xabbuh) [inline components/security/introduction.rst]: Should be indented by four spaces.

Comment (xabbuh) [inline components/security/introduction.rst]: And does the definition list render properly without blank lines separating the entries?

Comment (javiereguiluz) [inline components/security/introduction.rst]: Fixed these issues. Thanks.

**Comment by xabbuh** [comment]
:+1:

**Comment by WouterJ** [comment]
Thank you Javier!

I've removed the notion of Symfony 2.4 (and just say "The Security component is split [...]") in d85691c as our versioning rules state that we should remove "new in 2.4" from the 2.6 branch (at the time 2.6.0 was released, 2.4.x reached eom).

**Pull request closed without merging**